webpack.config.preact.test.js

```diff
@@ -1,35 +1,8 @@
 const path = require('path');
+const testBase = require('./webpack.config.test-base');
 
 module.exports = {
-  devtool: 'inline-source-map', // just do inline source maps instead of the default
```

Comment:
I likes the old formatting way more^^

Reply:
me too :(

this was done to try to make codeclimate happy